packages/devtools_server/lib/src/external_handlers.dart

```diff
@@ -27,28 +27,32 @@ import 'server_api.dart';
 /// DevTools project.
 Future<shelf.Handler> defaultHandler(
   ClientManager clients, {
-  String customDevToolsPath,
+  String? customDevToolsPath,
   bool debugMode = false,
 }) async {
-  String buildDir = customDevToolsPath;
+  String? buildDir = customDevToolsPath;
```

Comment:
nit: rewrite s
final buildDir = customDevToolsPath ?? await _resolveBuildDir() to eliminate the null checks.

Reply:
fixed